src/client/config.ts

```diff
@@ -17,5 +19,20 @@ export namespace BrokerConfigs {
      * @see https://kafka.apache.org/documentation/#brokerconfigs_auto.create.topics.enable
      */
     export const AUTO_CREATE_TOPIC_ENABLE = 'auto.create.topics.enable';
-
-}
+    
+    export async function isAutoCreateTopicEnabled(client: Client) : Promise<boolean> {
```

Comment:
it's not true/false, but enabled/disabled/unknown

Reply:
fixed